src/Featurizers/NormalizeFeaturizer.h

```diff
@@ -181,55 +199,55 @@ class NormalizeEstimator :
     using BaseType =
         Components::PipelineExecutionEstimatorImpl<
             Components::VectorNormsEstimator<IteratorRangeT, UpdaterT, MaxNumTrainingItemsV>,
-            Details::NormalizeEstimatorImpl<IteratorRangeT, UpdaterT, MaxNumTrainingItemsV>
+            Details::NormalizeEstimatorImplx<IteratorRangeT, UpdaterT, MaxNumTrainingItemsV>
         >;
 
     // ----------------------------------------------------------------------
     // |
     // |  Public Methods
     // |
     // ----------------------------------------------------------------------
-    NormalizeEstimator(AnnotationMapsPtr pAllColumnAnnotations, size_t colIndex);
-    ~NormalizeEstimator(void) override = default;
+    NormalizeEstimatorImpl(AnnotationMapsPtr pAllColumnAnnotations, size_t colIndex);
+    ~NormalizeEstimatorImpl(void) override = default;
 
-    FEATURIZER_MOVE_CONSTRUCTOR_ONLY(NormalizeEstimator);
+    FEATURIZER_MOVE_CONSTRUCTOR_ONLY(NormalizeEstimatorImpl);
 };
 
 /////////////////////////////////////////////////////////////////////////
 ///  \typedef       L1NormalizeEstimator
-///  \brief         Used to differentiate different NormalizeEstimator depends on norms
+///  \brief         Used to differentiate different NormalizeEstimatorImpl depends on norms

```

Comment:
Update this comment so that it captures information from the user's perspective.